src/Data/Nat/Divisibility.agda

```diff
@@ -300,3 +300,13 @@ m≤n⇒m!∣n! m≤n = help (≤⇒≤′ m≤n)
   help : ∀ {m n} → m ≤′ n → m ! ∣ n !
   help {m} {n}     ≤′-refl        = ∣-refl
   help {m} {suc n} (≤′-step m≤′n) = ∣n⇒∣m*n (suc n) (help m≤′n)
+
+------------------------------------------------------------------------
+-- Properties of quotient
+
+quotient∣n : ∀ {m n} (m∣n : m ∣ n) → quotient m∣n ∣ n
+quotient∣n {m = m} {n = n} m∣n = divides m $ begin-equality
```

Comment:
No idea how we didn't have this one yet. It should definitely go right at the top of the file?

Reply:
It is/was issue #1711 ...

Reply:
I've moved this to the top of the file. I don't want to close #1711 with just this, there are other relevant properties

Reply:
See  #2182 .